test/tests/functional/pbs_server_periodic_hook.py

```diff
@@ -106,7 +108,11 @@ def check_next_occurances(self, count, freq,
                                             interval=(hook_run_time + 1),
                                             starttime=search_after)
                 time_logged = self.get_timestamp(msg[1])
-                self.assertFalse((time_logged - time_expected) > 1)
+                if alarm != 0:
+                        self.assertLess((time_logged - time_expected),
+                                        (alarm - hook_run_time))
```

Comment:
I don't think the two extra set of () are needed.  They are two different arguments into the assert function.  There can't be any confusion on what is going on.